README.md

```diff
@@ -8,18 +8,25 @@ The automatically generated website can be viewed at https://iroboteducation.git
 Install required dependencies:
 
 ```bash
+sudo apt-get update
+sudo apt-get -y install python3-pip
+sudo apt-get -y install mkdocs
```

Comment:
No need for this as it's equivalent to the line below.

Reply:
Reverted this change in commit 8259333